modules/textures/src/compressed-texture-loader.js

```diff
@@ -1,10 +1,10 @@
 /** @typedef {import('@loaders.gl/loader-utils').LoaderObject} LoaderObject */
 /** @typedef {import('@loaders.gl/loader-utils').WorkerLoaderObject} WorkerLoaderObject */
-// Uncomment this line when latest version will be updated to '3.0.0-alpha.4'
+// Uncomment this line when latest version will be updated to '3.0.0-alpha.5'
 // import {VERSION} from './lib/utils/version';
 import {parseCompressedTexture} from './lib/parsers/parse-compressed-texture';
 
-const VERSION = '3.0.0-alpha.4';
+const VERSION = '3.0.0-alpha.5';
```

Comment:
Can't we drop this hack now that the worker is published and just use the actual version? No other workers need this hack.

Reply:
On the website it requests for the `latest` version. So I think we still need this hack till "3.0.0" is beta version.

Reply:
I see. We could probably be able to get it to request `beta` version instead of `latest`. 

Anyway this is good for now.